tests/common.py

```diff
@@ -177,6 +177,21 @@ def skip_onnxruntime_backend(message=""):
     return unittest.skipIf(config.is_onnxruntime_backend, reason)
 
 
+def check_onnxruntime_backend(message=""):
+    """ Skip if backend is onnxruntime """
+    config = get_test_config()
+    reason = _append_message("only supported by onnxruntime", message)
+    return unittest.skipIf(not config.is_onnxruntime_backend, reason)
+
+
+def check_onnxruntime_min_version(min_required_version, message=""):
+    """ Skip if onnxruntime version < min_required_version """
+    config = get_test_config()
+    reason = _append_message("conversion requires onnxruntime >= {}".format(min_required_version), message)
+    return unittest.skipIf(config.is_onnxruntime_backend and
+                           config.backend_version < LooseVersion(min_required_version), reason)
```

Comment:
skip if backend is NOT onnxruntime OR onnxruntime version < min_required_version?
so no need for check_onnxruntime_backend too.

Reply:
after discussion, decided to keep current semantics, similar to ```check_onnxruntime_incompatibility```